Packs/ExceedLMS/Integrations/IAMExceedLMS/IAMExceedLMS.py

```diff
@@ -0,0 +1,355 @@
+        :rtype: ``Optional[IAMUserAppData]``
+        """
+        uri = '/api/v2/users.json/'
+        params = {'api_key': self.api_key,
+                  'user[email]': email}
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri,
+            params=params
+        )
+        if isinstance(res, dict):
+            res = [res]
+        if res and len(res) == 1:
+            user_app_data = res[0]
+            user_id = user_app_data.get('id')
+            is_active = user_app_data.get('is_active')
+            username = user_app_data.get('login')
+
+            return IAMUserAppData(user_id, username, is_active, user_app_data)
+        return None
+
+    def create_user(self, user_data: Dict[str, Any]) -> IAMUserAppData:
+        """ Creates a user in the application using REST API.
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the created user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = '/api/v2/users.json'
+        params = {'api_key': self.api_key}
+        manager_id = self.get_manager_id(user_data)
+        if manager_id:
+            user_data['manager_id'] = manager_id
+            if 'manager_email' in user_data:
+                del user_data['manager_email']
+        user_app_data = self._http_request(
+            method='POST',
+            url_suffix=uri,
+            json_data=user_data,
+            params=params
+        )
+        user_id = user_app_data.get('id')
+        is_active = user_app_data.get('is_active')
+        username = user_app_data.get('login')
+        return IAMUserAppData(user_id, username, is_active, user_app_data)
+
+    def update_user(self, user_id: str, user_data: Dict[str, Any]) -> IAMUserAppData:
+        """ Updates a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the updated user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        demisto.results(str(user_id))
+        uri = f'/api/v2/users/{user_id}'
+        params = {'api_key': self.api_key}
+        manager_id = self.get_manager_id(user_data)
+        if manager_id:
+            user_data['manager_id'] = manager_id
+            if 'manager_email' in user_data:
+                del user_data['manager_email']
+        self._http_request(
+            method='PATCH',
+            url_suffix=uri,
+            json_data=user_data,
+            params=params,
+            resp_type='response'
+        )
+
+        username = user_data.get('login')
+
+        return IAMUserAppData(user_id=user_id, username=username, is_active=None, app_data=user_data)
```

Comment:
```suggestion
        return IAMUserAppData(user_id=user_id, username=username, is_active=True, app_data=user_data)
```
let's make it True, as created users are automatically active in this app

Reply:
This is the update function (and not the create user function) and we shouldn't hardcode the activity status.